_health-care/_js/reducers/veteran/index.js

```diff
@@ -0,0 +1,190 @@
+    isVaServiceConnected: false,
+    compensableVaServiceConnected: false,
+    receivesVaPension: false
+  },
+
+  additionalInformation: {
+    isEssentialAcaCoverage: false,
+    facilityState: '',
+    vaMedicalFacility: '',
+    wantsInitialVaContact: false
+  },
+
+  demographicInformation: {
+    isSpanishHispanicLatino: false,
+    isAmericanIndianOrAlaskanNative: false,
+    isBlackOrAfricanAmerican: false,
+    isNativeHawaiianOrOtherPacificIslander: false,
+    isAsian: false,
+    isWhite: false
+  },
+
+  veteranAddress: {
+    address: {
+      street: null,
+      city: null,
+      country: null,
+      state: null,
+      zipcode: null,
+    },
+    county: null,
+    email: null,
+    emailConfirmation: null,
+    homePhone: null,
+    mobilePhone: null
+  },
+
+  financialDisclosure: {
+    provideFinancialInfo: false,
+    understandsFinancialDisclosure: false
+  },
+
+  spouseInformation: {
+    spouseFullName: {
+      first: null,
+      middle: null,
+      last: null,
+      suffix: null,
+    },
+    spouseSocialSecurityNumber: null,
+    spouseDateOfBirth: {
+      month: null,
+      day: null,
+      year: null,
+    },
+    dateOfMarriage: {
+      month: null,
+      day: null,
+      year: null
+    },
+    sameAddress: false,
+    cohabitedLastYear: false,
+    provideSupportLastYear: false,
+    spouseAddress: {
+      street: null,
+      city: null,
+      country: null,
+      state: null,
+      zipcode: null,
+    },
+    spousePhone: null
+  },
+
+  childInformation: {
+    hasChildrenToReport: false,
+    children: []
+  },
+
+  annualIncome: {
+    veteranGrossIncome: null,
+    veteranNetIncome: null,
+    veteranOtherIncome: null,
+    spouseGrossIncome: null,
+    spouseNetIncome: null,
+    spouseOtherIncome: null,
+    childrenGrossIncome: null,
+    childrenNetIncome: null,
+    childrenOtherIncome: null
+  },
+
+  deductibleExpenses: {
+    deductibleMedicalExpenses: null,
+    deductibleFuneralExpenses: null,
+    deductibleEducationExpenses: null
+  },
+
+  insuranceInformation: {
+    isCoveredByHealthInsurance: false,
+    providers: []
+  },
+
+  medicareMedicaid: {
+    isMedicaidEligible: false,
+    isEnrolledMedicarePartA: false,
+    medicarePartAEffectiveDate: {
+      month: null,
+      day: null,
+      year: null
+    }
+  },
+
+  serviceInformation: {
+    lastServiceBranch: null,
+    lastEntryDate: {
+      month: null,
+      day: null,
+      year: null
+    },
+    lastDischargeDate: {
+      month: null,
+      day: null,
+      year: null
+    },
+    dischargeType: null
+  },
+
+  militaryAdditionalInfo: {
+    purpleHeartRecipient: false,
+    isFormerPow: false,
+    postNov111998Combat: false,
+    disabledInLineOfDuty: false,
+    swAsiaCombat: false,
+    vietnamService: false,
+    exposedToRadiation: false,
+    radiumTreatments: false,
+    campLejeune: false
+  }
+};
+
+function veteran(state = blankVeteran, action) {
+  let newState = undefined;
```

Comment:
I *think* you can. My preference is to be super explicit with these things, but if it bugs you, feel free to change it before merge.